libs/config/path_string_test.go

```diff
@@ -0,0 +1,84 @@
+package config_test
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/databricks/cli/libs/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewPathFromString(t *testing.T) {
+	for _, tc := range []struct {
+		input  string
+		output Path
+		err    error
+	}{
+		{
+			input:  "",
+			output: NewPath(),
+		},
+		{
+			input:  ".",
+			output: NewPath(),
+		},
+		{
+			input:  "foo.bar",
+			output: NewPath(Key("foo"), Key("bar")),
+		},
+		{
+			input:  "[1]",
+			output: NewPath(Index(1)),
+		},
+		{
+			input:  "foo[1].bar",
+			output: NewPath(Key("foo"), Index(1), Key("bar")),
+		},
+		{
+			input:  "foo.bar[1]",
+			output: NewPath(Key("foo"), Key("bar"), Index(1)),
+		},
+		{
+			input:  "foo.bar[1][2]",
```

Comment:
Can we have a path like `foo.bar[1][2][3]`?

Reply:
Yes, added a test case.